src/lit-element.ts

```diff
@@ -174,7 +171,7 @@ export class LitElement extends UpdatingElement {
   }
 
   /**
-   * Applies styling to the element shadowRoot using the [[`styles`]]
+   * Applies styling to the element shadowRoot using the `static get styles`
```

Comment:
I'd remove the "get" part here. We read the `static styles` property. In can incidentally by implemented with a geter.

Reply:
Also, please leave the brackets (`[[]]`) in place around `styles`--this creates a link in our Typedoc API docs.

Reply:
Fixed both. I think removing the brackets was a bad merge on my part.